gradle.properties

```diff
@@ -1,4 +1,4 @@
-version=29.63.2
+version=29.63.3
```

Comment:
this should be 29.64.0, so that apps that only bumped container but not restli will get a build error.

Reply:
OK makes sense